iocore/hostdb/HostDB.cc

```diff
@@ -77,6 +77,13 @@ HostDBCache hostDB;
 
 void ParseHostFile(const char *path, unsigned int interval);
 
+template <typename T>
+inline T
+CAP(T const &min, T const &max, T const &value)
```

Comment:
I think it would be better to add std::clamp to ink_std_compat.h and then use std::clamp in the code vs CAP.

reference: http://en.cppreference.com/w/cpp/algorithm/clamp

Reply:
Yes. I have another PR I'm working on that needs a C14 feature, so I put it in `lib/ts/c14_ulitity.h`. I'd put this in `lib/ts/c17_utility.h`.